spec/order_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "./lib/order.rb"
+
+describe Order do
+  subject(:order) {described_class.new}
+  let(:item1) {double(:item1)}
+  let(:item2) {double(:item2)}
+
+  before do
+      allow(item1).to receive(:name)       {"Salmon roll"}
+      allow(item1).to receive(:price)      {7.0}
+      allow(item2).to receive(:name)       {"Tuna roll"}
+      allow(item2).to receive(:price)      {8.5}
+  end
+
+  describe "#initialize" do
+
+    it "creates a blank array to store the ordered items in" do
+       expect(order.check_order.length).to eq 0
+     end
```

Comment:
Lint/BlockAlignment: `end` at 19, 5 is not aligned with `it "creates a blank array to store the ordered items in" do` at 17, 4.